src/pages/version/Tabs.tsx

```diff
@@ -17,7 +17,7 @@ const { parseQueryString } = queryString;
 interface Props {
   taskCount: number;
   isPatch: boolean;
-  childPatches: Partial<Patch>[];
+  childPatches: VersionQuery["version"]["patch"]["childPatches"];
```

Comment:
Thank you! I was never truly a fan of this usage of Partial since we don't actually know whats in the query that this is passed into.